src/css/editor-css.js

```diff
@@ -455,6 +455,13 @@ module.exports = `
     font-weight: normal;
     letter-spacing: normal;
     pointer-events: none;
+    max-height: 150px;
+    overflow-y: auto;
+    overflow-x: hidden;
+}
+
+.ace_tooltip * {
+    text-wrap: wrap;    
```

Comment:
Browser support for `text-wrap` seems [limited](https://caniuse.com/?search=text-wrap), can we get the same behavior in a way with better browser support?

Reply:
I'd say this rule should be removed white-space: pre; in the rule before should be changed to pre-wrap

Reply:
Unfortunately it wouldn't help with cases when non-breakable text is inside `pre` tag, because modern browsers (at least Chrome is doing that) are providing `text-wrap: nowrap;` for `pre` tag by default

Reply:
[screen-capture (33).webm](https://github.com/ajaxorg/ace/assets/32402726/b2458c5d-6535-42d2-9e3e-efa0efc15081)

Reply:
@mkslanc can we have both of the options above combined, so if we are in an older browser `white-space: pre-wrap` would do the trick?